app/services/region_backfill.rb

```diff
@@ -90,12 +90,8 @@ def find_or_create_region_from(parent:, region_type:, name: nil, source: nil)
     return existing_region if existing_region
 
     region = DryRunRegion.new(Region.new(name: region_name, region_type: region_type), dry_run: dry_run?, logger: logger)
-    if source
-      region.slug = source.slug
```

Comment:
Why wouldn't we want to copy over the source slug? This will keep some things simpler from a dashboard reports perspective.

Reply:
This is causing problems due to bad data in sandbox/staging. Some FGs have facilities in multiple states which breaks the slug uniqueness and those regions don't get created. We already have the `source` to map, is it necessary to match slugs? The other way is to make sure the data is extremely clean (hierarchically speaking) everywhere.